nautobot_circuit_maintenance/tests/test_handler.py

```diff
@@ -193,18 +201,18 @@ def test_run_invalid_notification(self):
     def test_process_raw_notification_no_provider_in_parser(self):
         """Test process_raw_notification with non existant Proivder in the parser library."""
         notification_data = get_base_notification_data()
-        test_notification = generate_raw_notification(notification_data, self.source.name)
+        test_notification = generate_email_notification(notification_data, self.source.name)
         test_notification.provider_type = "abc"
         res = process_raw_notification(self.job, test_notification)
-        self.assertEqual(res, None)
+        self.assertNotEqual(res, None)
```

Comment:
Why did this change? Are we now successfully processing the raw notification even though it's from an unknown provider_type?

Reply:
I'm still thinking on the final logic, this was just a first try on the change, but the `process_raw_notification` now creates the `RawNotification` even it can't be parsed (in this case because the parser library has no `provider`) 
Obviously this is not the final word, still in `draft` mode